dev-libs/libclc/libclc-0.2.0_pre20180610.ebuild

```diff
@@ -63,10 +64,18 @@ pkg_setup() {
 }
 
 src_configure() {
+	local libclc_targets=()
+
+	use video_cards_nvidia && libclc_targets+=("nvptx-- nvptx64-- nvptx--nvidiacl nvptx64--nvidiacl")
```

Comment:
The whole point of using an array is to use word splitting and quoting correctly, not put everything as one big item and then split it.